components/uid.rst

```diff
@@ -0,0 +1,155 @@
+.. index::
+   single: UID
+   single: Components; UID
+
+The UID Component
+=================
+
+    The UID component provides utilities to work with `unique identifiers`_ (UIDs)
+    such as UUIDs and ULIDs.
+
+.. versionadded:: 5.1
+
+    The UID component was introduced in Symfony 5.1 as an
+    :doc:`experimental feature </contributing/code/experimental>`.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/uid
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+This component provides different classes to generate each type of UID. The
+following sections explain each of them.
+
+UUIDs
+-----
```

Comment:
This should be `~` I think (same for all sections below)

Reply:
I disagree with this proposal because the TOC will look better and will be easier to navigate if we assign a high-level heading to each UID type. They are related but completely different, so users must find each one quickly and easily.

Reply:
Agreed, but then we should remove the  "usage" section imho.